model-client/src/commonMain/kotlin/org/modelix/model/client2/ReplicatedModel.kt

```diff
@@ -21,6 +21,9 @@ import org.modelix.model.lazy.CLTree
 import org.modelix.model.lazy.CLVersion
 import org.modelix.model.operations.OTBranch
 
+/**
+ * Dispose should be called on this, as otherwise a regular polling will go on.
+ */
 class ReplicatedModel(
```

Comment:
Can't we make ReplicatedModel implement Closeable for being able to [`use`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/use.html) it. That would make resource management a lot safer.

Reply:
That's a good idea. I created [MODELIX-568](https://issues.modelix.org/issue/MODELIX-568/) for this. We also did sth similar for the ModelClient itself in https://github.com/modelix/modelix.core/pull/235